vms/pom/pages/changePasswordPage.py

```diff
@@ -0,0 +1,30 @@
+# local Django
+from pom.pages.basePage import BasePage
+from pom.locators.changePasswordPageLocators import ChangePasswordPageLocators
+from pom.pages.homePage import HomePage
+
+
+class ChangePasswordPage(BasePage):
+    def __init__(self, driver):
+        self.driver = driver
+        self.change_password_page = ChangePasswordPage(self.driver)
```

Comment:
It's calling the constructor again and again leading to infinite recursion. Please correct this.